### PR TITLE
Bugfix/bsc1182106 clamsap setting

### PR DESCRIPTION
**Pull request by tomschr**
This PR fixes bsc1182106 and contains:

* A procedure to change the default virus definition for ClamSAP
* Use `INITDRIVERDIRECTORY`

See below a screenshot.

----

clampsap-p1
clampsap-p2

### CODE REVIEW

**Comment by tomschr** [comment]
@scmschmidt Sören, could you have a look if this makes sense? Thanks! :+1:

**Comment by scmschmidt** [comment]
Honestly I don't know. I have never done this and I don't have an installation to test it.

**Comment by tomschr** [comment]
No problem Sören. Thanks for your feedback. :+1:

File: xml/s4s_clamsap.xml [hunk 1]
```diff
@@ -250,6 +250,102 @@ Active: active (running) since Tue 2017-04-11 10:33:03 UTC; 24h ago
    </step>
   </procedure>
  </sect1>
+ <sect1 xml:id="sec-clamsap-changedir">
+  <title>Changing default localation of virus definition</title>
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/s4s_clamsap.xml]: Thanks! :+1:

**Comment by tomschr** [comment]
No feedback from customer, so it's save to merge it.

**Pull request merged**